File: cpp/zadanie10.cpp
#include <iostream>
#include <fstream>
#include <string>

int main() {
	std::fstream inFile;
	std::fstream outFile;

	inFile.open("dane.txt", std::ios::in);
	outFile.open("wynik.txt", std::ios::out);
	
	int date;

	while (!inFile.eof()) {
		date = 0;
		inFile >> date;
		if (!(date % 2))
			outFile << date << std::endl; 
	}

	inFile.close();
	outFile.close();

	return 0;
}

File: javascript/zadanie10.js
var fs = require('fs')

var file = fs.readFileSync('dane.txt').toString('utf8')
var dates = file.split('\r\n')

if (fs.existsSync('wynik.txt'))
    fs.unlinkSync('wynik.txt')

dates.forEach(date => {
    if (!(date % 2))
        fs.appendFileSync('wynik.txt', date + '\r\n')
})